src/_P005_DHT.ino

```diff
@@ -187,7 +187,7 @@ bool P005_do_plugin_read(struct EventStruct *event) {
   byte dht_dat[5];
   for (i = 0; i < 5 && !readingAborted; i++)
   {
-      byte data = Plugin_005_read_dht_dat();
+      int data = Plugin_005_read_dht_dat();
```

Comment:
I don't think this will make a difference here, since the result variable is a byte to begin with and the value is just a set of 8 bits:

```c++
/*********************************************************************************************\
* DHT sub to get an 8 bit value from the receiving bitstream
\*********************************************************************************************/
int Plugin_005_read_dht_dat(void)
{
  byte i = 0;
  byte result = 0;
  for (i = 0; i < 8; i++)
  {
    if (!P005_waitState(1))  return -1;
    delayMicroseconds(35); // was 30
    if (digitalRead(Plugin_005_DHT_Pin))
      result |= (1 << (7 - i));
    if (!P005_waitState(0))  return -1;
  }
  return result;
}
```

Reply:
in line 191 there is a check for returned -1

Reply:
@TD-er is it ok now?

Reply:
OK, missed the -1 check.
Good find :)

Reply:
so it is ready to merge, just find time ;)